### ISSUE TITLE
CRAM sequences inside variables

### ISSUE THREAD

**Issue by elydpg** [alda-lang/alda-core#55]
Consider the following:

```
(tempo! 90)
cram = {f+14~/a+~ a+~/b~ f+1~/a+~}2
piano: {f+14~/a+~ a+~/b~ f+1~/a+~}2
#plays normally
piano: cram
#this should play identically to the previous line, but it doesn't. For some reason the cram being stored in the variable trips it up.
```

**Comment by daveyarwood** [comment]
I can confirm that this is a bug -- just haven't had time to look into it yet.

I'll get to it soon, unless someone else feels like jumping in ;)

**Comment by elydpg** [comment]
still a bug in 1.0.0rc77

**Comment by daveyarwood** [comment]
Fixed in [1.0.0-rc78](https://github.com/alda-lang/alda/releases/tag/1.0.0-rc78).
